app/views/listings/form/_price.haml

```diff
@@ -33,7 +33,7 @@
                 "#fee_display",
                 "#listing_currency",
                 #{commission_from_seller},
-                #{minimum_commission},
+                #{minimum_commission.format(decimal_mark: ".", symbol: false)},
```

Comment:
...and I just realized that this value is probably passed to JS as an integer and thus my comment should be ignored.